app/main.py

```
@@ -308,6 +308,48 @@ async def startup_event():
+
+    # Clear rate limiters and stop cleanup tasks
+    try:
+        from app.core.rate_limiting.memory_limiter import get_rate_limiter, get_gemini_limiter
+        rate_limiter = get_rate_limiter()
+        if hasattr(rate_limiter, 'stop'):
+            await rate_limiter.stop()
+        gemini_limiter = get_gemini_limiter()
+        if hasattr(gemini_limiter, 'stop'):
+            await gemini_limiter.stop()
+        logging.info("Rate limiters stopped")
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)